object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionTransfer.java

```diff
@@ -167,9 +227,41 @@ public void accumulate(Node node, TransferResult<CFValue, CFStore> result, Strin
           }
         }
       }
+      AnnotationMirror newAnno = atypefactory.createAccumulatorAnnotation(valuesAsList);
+      exceptionalStores.values().stream().forEach(s -> s.insertValue(target, newAnno));
     }
-    AnnotationMirror newAnno = atypefactory.createAccumulatorAnnotation(valuesAsList);
-    exceptionalStores.values().stream().forEach(s -> s.insertValue(target, newAnno));
+  }
+
+  private void insertIntoStores(
```

Comment:
Is this method copy-pasted from somewhere else?

Reply:
Yes, It's from AccumulationTransfer class.

Reply:
can we find a way to not do copy-paste?

Reply:
Can we make this method public in AccumulationTransfer?

Reply:
@Nargeshdb we can make it `protected` and then call it from here

Reply:
This can be a follow up change as it requires a change in CF